syntax = "proto3";
package custom;




import "cmdb_extend_sdk/model/cmdb_extend/system_dependency.proto";

/*
GetAllSystemTree返回
*/
message GetAllSystemTreeResponse {
    /*
    业务树
    */
    repeated cmdb_extend.SystemDependency systems = 1 ;
}

/*
GetAllSystemTreeApi返回
*/
message GetAllSystemTreeResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    custom.GetAllSystemTreeResponse data = 4 ;
}

